CONTRIBUTING.md

```diff
@@ -25,10 +34,45 @@ This project uses GitHub Actions to publish the packages automatically to npm. N
 ### Steps to create a new package version
 
 1. Merge all appropriate PRs you want to publish into main branch
-1. Run the `make version` command to bump the version number in packages (new version number is determined automatically based on commit history)
-1. Check that the version number is correct and everything looks good
-1. Run manually `git push && git push --tags` to push the changes to remote
-1. Publishing is done automatically by GitHub Actions (uses `build` script and `make publish` command)
+2. Run the `make version` command to bump the version number in packages (new version number is determined automatically based on commit history)
+3. Check that the version number is correct and everything looks good
+4. Run manually `git push && git push --tags` to push the changes to remote
+5. Publishing is done automatically by GitHub Actions (uses `build` script and `make publish` command)
+
+### Steps for package development
+
+#### Web
+
+1. Component root
+   - Scripts
```

Comment:
I will prefer to use `JS plugins`, because `scripts` are mainly connected with npm scripts in this repository.